### DISCUSSION TITLE
建议

### DISCUSSION THREAD

**Discussion by Juhnkerg** [lencx/Noi#109] [Ideas]
- 提示命令的窗口有点小，建议添加可以上下滑动改变窗口大小的功能
- 希望能添加Mac上command+w快捷键关闭窗口，但不关闭app

**Comment by lencx** [comment]
1. 提示窗口有 3 个 size 可以调整，在窗口的右上角有个 s、m、l 的图标。它不支持自定义拖拽大小，因为这里会牵扯到复杂的布局计算。
2. 在菜单中有个“点击 X 退出”，可以将它取消勾选，这样就不会因为点击 X 退出应用了。唤起/隐藏应用可以在设置中的快捷键中配置一下全局快捷键。
